examples/p-xylene-explicit/setup/setup.leap.in

```diff
@@ -0,0 +1,36 @@
+# Create a benzene-toluene system.
```

Comment:
This entire file should be deleted, because none of its outputs are used anymore.